Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jamulus/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/es/1-index.po:
```diff
@@ -128,19 +107,19 @@ msgstr "¿Quieres involucrarte?"
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribución) para saber cómo. ¡Cualquier persona es bienvenida!"
```
Not sure if that's the correct link:

```suggestion
msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribution) para saber cómo. ¡Cualquier persona es bienvenida!"
```

ignotus666 replied:
Thanks - corrected. There's a user on Weblate who has the exasperating habit of changing stuff that was already ok. He's well-intentioned but it's exhausting, as I have to go after him and comb through everything he touches... I've repeatedly told him to leave links alone, to not arbitrarily change things on a whim without any consistency... If it keeps going on I'll have to end up blocking him.

ann0see commented on _translator-files/po/de/1-index.po:
```diff
@@ -79,10 +64,9 @@ msgstr "Was ist Jamulus?"
+#, no-wrap
 msgid "Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)\n"
-msgstr "       Mit Jamulus kannst du mit deinen Freunden, deiner Band oder jedem, den du online findest, spielen, proben oder jammen. Spiele aus der Ferne im Takt mit hochqualitativem Sound mit geringer Latenz über eine normale Breitbandverbindung. "
+msgstr "Jamulus ermöglicht es dir, mit deinen Freunden, deiner Band oder jedem, den du online findest, zu spielen, zu proben oder gemeinsam zu musizieren. Spiele zusammen in Echtzeit mit hochwertigem, geringem Latenz-Sound über eine normale Breitbandverbindung. [Lade es hier herunter!](wiki/Getting-Started)\n"
```
This needs some thought.

ann0see commented on _translator-files/po/de/1-index.po:
```diff
@@ -99,9 +83,8 @@ msgstr "Jamulus weltweit"
 msgid "All over the world Jamulus allows choirs to rehearse and rock bands to play. Jamulus brings folk and classical musicians together. It's being used for remote music lessons, in schools and universities, in private and in public — all in real time on the Internet, as if you were there in person.\n"
 msgstr ""
-"    Überall auf der Welt können dank Jamulus Chöre zusammen proben und Rockbands ihre Gigs aufführen.\n"
+"Überall auf der Welt, können dank Jamulus Chöre zusammen proben und Rockbands ihre Gigs aufführen.\n"
```
No. That's wrong -> revert

ann0see reviewed (changes requested):
Unfortunately German needs updates.

ann0see commented on _translator-files/po/nl/1-index.po:
```diff
@@ -116,6 +117,8 @@ msgid ""
 "Check out the [documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!\n"
 "You can also ask on the [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 msgstr ""
+"Lees de [documentatie](wiki/Getting-Started) en overweeg ook nog een bezoek aan de [troubleshooting sectie](wiki/Client-Troubleshooting)!\n"
```
@henkdegroot please check if that's ok.

henkdegroot replied:
Looks okay to me.

ann0see replied:
Thanks for the confirmation

ann0see commented on _translator-files/po/it/1-index.po:
```diff
@@ -82,10 +83,9 @@ msgstr "Cos'è Jamulus?"
+#, no-wrap
 msgid "Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)\n"
-msgstr "       Jamulus ti consente di suonare, provare o suonare con i tuoi amici, la tua band o chiunque trovi online. Suona insieme da remoto a tempo con audio di alta qualità e bassa latenza su una normale connessione a banda larga. "
+msgstr "Jamulus ti consente di suonare, provare o suonare con i tuoi amici, la tua band o chiunque trovi online. Suona insieme da remoto a tempo con audio di alta qualità e bassa latenza su una normale connessione a banda larga. [Scaricalo qui!](wiki/Getting-started)\n"
```
Fixed a link. Should still be checked.

weblate committed:
Update Spanish translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ignotus <ignotus666@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/es/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Startpage

weblate committed:
Update German translation using Weblate

Currently translated at 66.6% (8 of 12 strings)

Update German translation using Weblate

Currently translated at 66.6% (8 of 12 strings)

Update German translation using Weblate

Currently translated at 66.6% (8 of 12 strings)

Update German translation using Weblate

Currently translated at 58.3% (7 of 12 strings)

Update German translation using Weblate

Currently translated at 58.3% (7 of 12 strings)

Update German translation using Weblate

Currently translated at 50.0% (6 of 12 strings)

Co-authored-by: Eric Brändli <ericbraendli@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Victor Jouhoff <jouhoff@pm.me>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/de/
Translation: Jamulus/Startpage

weblate committed:
Update Norwegian Bokmål translation using Weblate

Currently translated at 58.3% (7 of 12 strings)

Co-authored-by: Allan Nordhøy <epost@anotheragency.no>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/nb_NO/
Translation: Jamulus/Startpage

weblate committed:
Update French translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Update French translation using Weblate

Currently translated at 100.0% (12 of 12 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update French translation using Weblate

Currently translated at 100.0% (159 of 159 strings)

Update French translation using Weblate

Currently translated at 75.0% (9 of 12 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Olivier Humbert <trebmuh@tuxfamily.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/fr/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/fr/
Translation: Jamulus/Running-a-Server
Translation: Jamulus/Startpage

weblate committed:
Update Italian translation using Weblate

Currently translated at 66.6% (8 of 12 strings)

Update Italian translation using Weblate

Currently translated at 66.6% (8 of 12 strings)

Co-authored-by: Emanuele Frasca <noostale@live.it>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/it/
Translation: Jamulus/Startpage

weblate committed:
Update Dutch translation using Weblate

Currently translated at 50.0% (6 of 12 strings)

Co-authored-by: Dklfajsjfi49wefklsf32 <nlincus@users.noreply.hosted.weblate.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/1-index/nl/
Translation: Jamulus/Startpage

ann0see reviewed (approved):
Seems to be ok, but I'd still like to have an ok from a few others.

trebmuh commented:
ok
